media/chat.css

```
@@ -45,6 +45,16 @@ body.vscode-dark .brand-logo, body.vscode-high-contrast .brand-logo { filter: in
 }
 .icon-button:hover, .text-button:hover { background: var(--vscode-toolbar-hoverBackground); }
 .icon-button.compact { width: 23px; height: 23px; }
+#fork,
+#import-session,
+#export-session {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 600;
+  line-height: 1;
+  -webkit-text-stroke: 0.18px currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)